mlrun/utils/helpers.py

```diff
@@ -245,6 +246,29 @@ def get_regex_list_as_string(regex_list: List) -> str:
     return "".join(["(?={regex})".format(regex=regex) for regex in regex_list]) + ".*$"
 
 
+def is_file_path_invalid(code_path, file_path):
+    """
+    The function checks if the given file_path is a valid path
+    If the file_path is a relative path, it is completed by joining it with the code_path.
+    Otherwise, the file_path is used as is.
+    Additionally, it checks if the resulting path exists as a file, unless the file_path is a remote URL.
+    If the file_path has no suffix, it is considered invalid.
+
+    """
+    if file_path.startswith("./") or (
+        "://" not in file_path and os.path.basename(file_path) == file_path
+    ):
+        abs_path = os.path.join(code_path, file_path.lstrip("./"))
+    else:
+        abs_path = file_path
+
+    return (
+        not file_path
```

Comment:
better check if not file_path at the top of the function because if the file_path is None then it will explode on `startswith`